cypress/integration/login.js

```diff
@@ -1,27 +1,108 @@
+import LoginPage from '../objects/loginPage'
+import Timelog from '../objects/timelog'
+
+const loginPage = new LoginPage();
+const timelog = new Timelog();

```

Comment:
according to camel case naming convention every new word in variable name should start with first capital letter
time**L**og